examples/kafka/kafka.json

```diff
@@ -0,0 +1,26 @@
+{
+  "jobName": "kafka",
+  "image": "openpai/pai.example.kafka:latest",

```

Comment:
>openpai [](start = 12, length = 7)

please include your dockerfile in this PR

Reply:
`latest` tag is not necessary